spec/unit/puppet/provider/ios_interface/cisco_ios_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+module Puppet::Provider::IosInterface; end
+require 'puppet/provider/ios_interface/cisco_ios'
+
+RSpec.describe Puppet::Provider::IosInterface::CiscoIos do
+  let(:utility) { described_class }
+
+  def self.load_test_data
+    PuppetX::CiscoIOS::Utility.load_yaml(File.expand_path(__dir__) + '/test_data.yaml', false)
+  end
+
+  it_behaves_like 'resources parsed from cli'
+  it_behaves_like 'commands created from instance'
+  # it_behaves_like 'a noop canonicalizer'
```

Comment:
Just remove this rather than commenting it out

Reply:
done